src/styles/components/_mastheads.scss

```diff
@@ -75,22 +75,20 @@
 
 .w-masthead-path__container {
   align-items: center;
-  box-sizing: content-box;
   display: flex;
-  flex-direction: column-reverse;
```

Comment:
did `-reverse` actually manifest in a serious way before?

Reply:
> did -reverse actually manifest in a serious way before?

it used to make the image go before the text on mobile. That was the original design but I changed it in this PR